src/request_parser.ts

```diff
@@ -47,6 +46,11 @@ export const parseTakerRequest = (req: Pick<express.Request, 'headers' | 'query'
             apiKey = undefined;
         }
 
+        let requestUuid = req.headers[ZERO_EX_REQUEST_UUID_HEADER_STRING];
+        if (typeof requestUuid !== 'string') {
+            requestUuid = undefined;
```

Comment:
Shouldn't we be enforcing this?

Reply:
I'm reusing this parser across both `/price` and `/rfqm/v2/price` so it's optional unfortunately